### PR TITLE
Feature/add language input to documents

### PR DESCRIPTION
**Pull request by kowal**
#### Summary

Add language input/label to
- document show page
- documents index page
- document form (in the litigation context)

Languages are taken from https://github.com/scsmith/language_list.

Zrzut ekranu 2019-08-11 o 17 37 21

Zrzut ekranu 2019-08-12 o 14 58 18

### CODE REVIEW

File: app/decorators/litigation_decorator.rb [hunk 1]
```diff
@@ -21,7 +21,10 @@ def document_links
     return [] if model.documents.empty?
 
     model.documents.map do |document|
-      h.link_to document.name, document.url, target: '_blank'
+      h.link_to document.name,
+                document.url,
+                target: '_blank',
+                title: document.external? ? document.url : nil
```
Comment (tsubik) [inline app/decorators/litigation_decorator.rb]: not sure why the title should be `document.url` here and not always `document.name`

Comment (kowal) [reply app/decorators/litigation_decorator.rb]: I guess it could be handy for users to see where those external links are pointing.

**Comment by kowal** [comment]
> I think you can add `language` to factory too

it was already there, but I'll add comment on what format to use

File: app/decorators/litigation_decorator.rb (hunk 1 above)
Comment (tsubik) [reply app/decorators/litigation_decorator.rb]: ahh, ok

**Pull request merged**